services/blockstorage/service.go

```diff
@@ -86,7 +86,7 @@ func (s *service) ValidateBlockForCommit(input *services.ValidateBlockForCommitI
 }
 
 func (s *service) RegisterConsensusBlocksHandler(handler handlers.ConsensusBlocksHandler) {
-	panic("Not implemented")
+	s.consensusBlocksHandlers = append(s.consensusBlocksHandlers, handler)
```

Comment:
@talkol Why did you need to implement this? At any point (now) there's only one consensus active.